src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/help/Get-AzureRmRecoveryServicesBackupStatus.md

```diff
@@ -0,0 +1,134 @@
+---
+external help file: Microsoft.Azure.Commands.RecoveryServices.Backup.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmRecoveryServicesBackupStatus
+
+## SYNOPSIS
+Checks whether your ARM resource is backed up or not.
+
+## SYNTAX
+
+### Name (Default)
+```
+Get-AzureRmRecoveryServicesBackupStatus -Name <String> -ResourceGroupName <String> -Type <String>
+ [-DefaultProfile <IAzureContextContainer>]
+```
+
+### Id
+```
+Get-AzureRmRecoveryServicesBackupStatus -ResourceId <String> [-DefaultProfile <IAzureContextContainer>]
+```
+
+## DESCRIPTION
+The command returns null/empty if the specified resource is not protected under any Recovery Services vault in the subscription. 
```

Comment:
@cormacpayne 
- The Vault object is present in the RecoveryServices module. As we can't have inter-module dependencies, we are returning vault IDs instead of the Vault object itself. I hope this answers your question.
- Regarding the name of the cmdlet, we have already addressed this in the design review (https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/82). Can you let us know what has changed your stance on this matter? @pvrk what do you think about Cormac's suggestion?

Reply:
@cormacpayne: The command is not to test whether a backup vault exists. It is to check whether a VM is already protected. If a VM is protected, some time we have seen users ask for other specific information like latest recovery point, policy etc for which getting the Id/name of the vault is mandatory. So, If a VM is protected, return the vault Id/object. If a VM is not protected, let the customer know (by returning empty value) and then he will proceed to protect it. So, I feel it should be "Get-......backupStatus". Please let me know your thoughts.

Reply:
@dragonfly91 @pvrk responded via email 😀